ENGAGEHF/Bluetooth/Views/MeasurementRecordedView.swift

```diff
@@ -0,0 +1,123 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SpeziViews
+import SwiftUI
+
+
+struct MeasurementHeader: View {
+    @Environment(\.dismiss) var dismiss
+    @Binding var viewState: ViewState
+    
+    
+    var body: some View {
+        ZStack {
+            HStack {
+                Button(action: {
+                    dismiss()
+                }) {
+                    Text("DISMISS_BUTTON")
+                        .foregroundStyle(Color.accentColor)
+                }
+                    .buttonStyle(PlainButtonStyle())
+                    .padding(.trailing, 10)
+                    .disabled(viewState != .idle)
+                    .accessibilityLabel("Dismiss")
+                
+                Spacer()
+            }
+            
+            HStack {
+                Spacer()
+                
+                Text("Measurement Recorded")
+                    .font(.title2)
+                
+                Spacer()
+            }
+        }
+            .padding()
+    }
+}
+
+
+struct MeasurementLayer: View {
+    private var measurement = MeasurementManager.manager.newMeasurement
+    private var textSize: CGFloat = 60
+    
+    
+    var body: some View {
+        Text(measurement?.quantity.description ?? "???")
+            .font(.system(size: textSize, weight: .bold, design: .rounded))
+    }
+}
+
+
+struct DiscardButton: View {
+    @Environment(\.dismiss) var dismiss
+    
+    
+    var body: some View {
+        Button(action: {
+            dismiss()
+        }) {
+            Text("DISCARD")
+                .foregroundStyle(Color.red)
+        }
```

Comment:
Applies to this button and some other views in the app: I am not a huge fan of this trailing closure formatting if we have multiple closures, I would suggest the following formatting:
```suggestion
        Button(
            action: {
                dismiss()
            },
            label: {
                Text("Discard")
                    .foregroundStyle(Color.red)
            }
        )
```